test-app/ember-cli-build.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function (defaults) {
+  let app = new EmberApp(defaults, {
+    autoImport: {
+      watchDependencies: ['ember-cli-flash'],
+    },
+  });
+
+  const { maybeEmbroider } = require('@embroider/test-setup');
+  return maybeEmbroider(app);
```

Comment:
Instead of maybe embroider, we courd do always strictest embroider. 
This would guarantee no accidental regressions are possible, and also provide a better debugging experience

Reply:
Good idea